ARCHITECTURE.md

```diff
@@ -0,0 +1,131 @@
+# OpenSearch Security Plugin Architecture
+
+OpenSearch’s core systems do not include security features, these features are added by installing the Security Plugin.  The Security Plugin accesses OpenSearch’s internals to build up Authentication, Authorization, End to End Encryption, Audit Logging, and management interfaces.

```

Comment:
```suggestion
OpenSearch’s core systems do not include security features, these features are added by installing the Security Plugin.  The Security Plugin accesses OpenSearch’s internals to build up authentication, authorization, end-to-end encryption, audit logging and management interfaces.

```

Reply:
I would also suggest swapping the second sentence to "The Security Plugin extends OpenSearch to provide..."